### ISSUE TITLE
Feature request: Delete empty log groups

### ISSUE THREAD

**Issue by vjsingh** [binxio/aws-cloudwatch-log-minder#22]
It would be nice to have a command to delete empty log groups. This seems useful after running `delete_empty_log_streams`.

The below seems to work for me (need to clean up)

```
def _delete_empty_log_groups(
    group: dict, purge_non_empty: bool = False, dry_run: bool = False
):
    now = datetime.utcnow().replace(hour=0, minute=0, second=0, microsecond=0)
    log_group_name = group["logGroupName"]
    retention_in_days = group.get("retentionInDays", 0)
    if not retention_in_days:
        log.info(
            "skipping log group %s as it has no retention period set",
            log_group_name,
        )
        return

    kwargs = {
        "logGroupName": log_group_name,
        "orderBy": "LastEventTime",
        "descending": False,
        "PaginationConfig": {"PageSize": 50},
    }

    for response in cw_logs.get_paginator("describe_log_streams").paginate(
        **kwargs
    ):
        if len(response["logStreams"]) == 0:
            log.info(
                "Deleting group %s", log_group_name,
            )
            cw_logs.delete_log_group(logGroupName=log_group_name)

```

Thanks for such a great python script!

**Comment by FriedCircuits** [comment]
@vjsingh Could you post a full copy of your working code to delete empty log groups?

**Comment by FriedCircuits** [comment]
Any chance of this getting merged in?

**Comment by mvanholsteijn** [comment]
Normally I  create the the log groups with CloudFormation so I will not delete them by hand. If you feel this is a useful feature, can you create a PR to include the option in the utility? I will be happy to review and merge.

**Comment by FriedCircuits** [comment]
I got it working with some other improvements and plan to do a PR tomorrow.

**Comment by mvanholsteijn** [comment]
solved in v0.5.1
